.gitpod.yml

```diff
@@ -13,7 +13,10 @@ ports:
 
 tasks:
   - name: Forem Server
-    before: redis-server &
+    before: >
+      redis-server &
+      gp await-port 5432 &&
+      sleep 1
```

Comment:
I hate everything about this. 

https://github.com/gitpod-io/workspace-images/blob/master/postgres/Dockerfile#L19-L22 it looks like postgres doesn't start until the bashrc is loaded (and this might not provide enough time).

One alternate might be to inline the bashrc magic to the before action here and ensure it completes:

```bash
pg_ctl status | grep PID) ]] || pg_start
```